README.md

```diff
@@ -68,14 +68,15 @@ This collection provides a list of educational resources for Software Engineers.
 
 > If you think there is a resource which you think others would benefit on, fork, and your changes, and make a pull request!
 
-### Here are just 5 from our huge range...
+### Here are just 6 from our huge range...
```

Comment:
Don't write 6 here. we only add 5 resources and add rest of the resources in resources/learnPython.md